test/ZZMatrix-linalg-test.jl

```diff
@@ -55,5 +55,12 @@ end
                   -21384 2805 -6662 2496 -2005 4244 25011 -26631 -14973 11667;
                   -2551 -491 -893 839 68 445 2865 -3356 -2666 2029]);
   @test is_unimodular(A^7)
+
+  # Smallest failing example I have found (sigh)
+  A = matrix(ZZ,[-322341980522719490138313379573816217155164761827 6969888613235234140467966547111722569499035362 128959401887831470883414283176720530849782945792;
+ -1457773803314294556050124560975280637500592484816 31520936292307671829042204400726647073448292204 583211772349057793876747339337640547820341591863;
+ 8600567719843461600845233922140566452220386100150 -185967086634780997501807202322245574319382421231 -3440830348092464486770252906732722774985068539150]);
+b = matrix(ZZ,3,1,[46, 72, 9]);
+x = Nemo.dixon_solve(A,b);
```

Comment:
```suggestion
  b = matrix(ZZ,3,1,[46, 72, 9]);
  x = Nemo.dixon_solve(A,b);
```
and maybe test that the result is correct?

Reply:
OK, it makes sense to check that the result is correct -- previously it simply gave an error.